css/style.css

```diff
@@ -1,14 +1,278 @@
-
- /* Use the 'Roboto' font from Google. Already added to your HTML header */
-
 body {
+    display: flex;
     font-family: 'Roboto', sans-serif;
+    flex-flow: column wrap;
+    font-size: 1.05em;
+    font-stretch: 100%;
+    font-style: normal;
     -webkit-font-smoothing: antialiased;
 }
 
-/* Add your custom styles here */
-/* Remember: be organised, use comments and separate your styles into meaningful chunks */
-/* For example: General styles, Navigation styles, Hero styles, Footer etc. */
+nav,
+main,
+header,
+footer,
+section,
+.container,
+ul,
+li,
+figure {
+    display: flex;
+}
+
+main {
+    flex-flow: column nowrap;
+    padding-top: 70px;
+}
+
+ul,
+.container {
+    align-items: center;
+}
+
+.container {
+    width: 58%;
+}
+
+ul {
+    flex-flow: row wrap;
+    justify-content: space-between;
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0;
+}
+
+figure {
+    margin: 0;
+}
+
+h1,
+h2,
+h3,
+h5,
+span {
+    font-weight: 300;
+    text-align: center;
+}
+
+h1 {
+    font-size: 3em;
+    margin: 0;
+}
+
+h2 {
+    font-size: 2.6em;
+    margin-top: 0;
+}
+
+h3 {
+    font-size: 1.5em;
+}
+
+h5 {
+    font-size: 1em;
+    margin-top: 0;
+}
+
+hr {
+    align-self: flex-start;
+    border: 0;
+    border-top: 1px solid #eaebec;
+    margin: 1.313rem 0;
+    width: 100%;
+}
+
+i {
+   font: normal normal normal 14px/1 FontAwesome;
+}
+
+:root {
+    --main-color: #f15a29;
+}
+
+
+/* Navigation Menu */
+
+nav {
+    background-color: white;
+    border-bottom: 1px solid #DDD;
+    justify-content: center;
+    min-height: 70px;
+    position: fixed;
+    width: 100%;
+}
+
+nav .container {
+    justify-content: space-between;
+}
```

Comment:
This looks good, though below 1200px it maybe causes the nav items to be a little too stretched away from each other? You could use `flex-end` and keep that alignment more consistent.